.github/workflows/linearizability-nightly.yaml

```diff
@@ -1,9 +1,12 @@
 name: Linearizability Nightly
+permissions: read-all
 on:
   # schedules always run against the main branch, hence we have to create separate jobs
   # with individual checkout actions for each of the active release branches
   schedule:
     - cron: '25 9 * * *' # runs every day at 09:25 UTC
+  push: {}
+  pull_request: {}
```

Comment:
not sure why adding this? `on.schedule` already defines the time to run the workflow, usually we don't need `on.push` or `on.pull_request` in this case.

Reply:
No this is temporary to make sure that the test pass.